File: Java/Java source-code/find_euc_dist.java


public class find_euc_Dist {
    public static double find_euc_Dist_m(Integer[] a, Integer[] b) {
        int i;
        double sum = 0;
        for (i = 0; i < a.length; i++) {
            sum += (a[i] - b[i]) * (a[i] - b[i]);
        }
        double result = Math.sqrt(sum);
        return result;
    }
}

File: C++/find_euc_dist.cpp
#include <iostream>

double find_euc_dist(int a[], int b[], int na) {
    double sum = 0;
    for (int i = 0; i < na; i++) {
        sum += (a[i] - b[i]) * (a[i] - b[i]);
    }
    double result = sqrt(sum);
    return result;
}